src/providers/ibmcloud/mod.rs

```diff
@@ -23,6 +24,10 @@ use crate::errors::*;
 use crate::network;
 use crate::providers::MetadataProvider;
 
+use mailparse::*;
+
```

Comment:
Nit: you can remove this newline and move all the `use` statements together (`rustfmt` will sort them properly).

Reply:
It looks like the Travis config is using `cargo fmt -- --check -l` and `cargo clippy -- -D warnings` and neither of those seemed to suggest changes to the `use` statements?  I might be looking at the wrong Travis config, or using the wrong version of the linters, or perhaps the wrong rulesets.  I removed this newline manually in the next commit.